app/apps/server/communication/rest.js

```diff
@@ -63,14 +111,45 @@ export class AppsRestApi {
 				let buff;
 
 				if (this.bodyParams.url) {
+					if (process.env.NODE_ENV === 'production') {
```

Comment:
@d-gubert Might need to update the CLI docs to instruct them put instance in developer mode by setting to set NODE_ENV=development environment variable for their instance.  Or run via meteor

Reply:
What is this disabling? The possibility of install apps via URL and upload in production?

Reply:
Correct, it disables installing apps in production via urls and via posting the app zip in the body. It helps disable a security risk for any production servers and ensures they install via a marketplace.

Reply:
I don't think we should disable the possibility of install own custom apps. May add a setting to for enable a "development mode" before allow this kind of installation.

Reply:
Yes, this particular `if` is disabling installation through URL for production envs. There is another check in the block that installs zips.

I'm not sure this is the best way to do this... maybe a setting enabling a "developer" mode in the Server is better? Imagine for instance a maintainer that makes an app that makes sense only to their company's context; they won't be able to install their app in their currently running servers because of this